gctf/__init__.py

```diff
@@ -60,15 +60,16 @@ def getEnviron(cls):
     @classmethod
     def getProgram(cls):
         """ Return the program binary that will be used. """
-        if (not GCTF in os.environ or
-            not GCTF_HOME in os.environ):
+        if (GCTF not in os.environ or
+            GCTF_HOME not in os.environ):
             return None
 
-        return os.path.join(os.environ[GCTF_HOME], 'bin',
+        return os.path.join(cls.getHome('bin'),
                             os.path.basename(os.environ[GCTF]))
 
     @classmethod
     def isNewVersion(cls):
-        return not  pyworkflow.em.Plugin.getActiveVersion().startswith("0.50")
+        return not cls.getActiveVersion().startswith("0.50")
```

Comment:
This will return False for version 1.06, is that fine?

Reply:
this will return True for 1.06/1.18

Reply:
You are right, maybe I missed the "not" :(